From 7cf56a809891b25b73607f4a4a1f36474dfb2743 Mon Sep 17 00:00:00 2001
From: Karsten Merker <merker@debian.org>
Date: Tue, 6 Jun 2017 23:08:57 +0200
Subject: [PATCH] Increase the partition size for the armhf SD card images
 (Closes: #864260).

The installer binary size has grown over time and now exceeds the
predefined SD card partition size for the netboot build on armhf.
The hd-media image just tightly fits for now but will probably
outgrow the available space in the near future as well.

Increase the partition size from ~40MB to ~50MB for both netboot
and hd-media.  As the images get compressed, the actual download
size increase for additional unused space is rather small.
---
 build/config/armhf/hd-media.cfg | 2 +-
 build/config/armhf/netboot.cfg  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/build/config/armhf/hd-media.cfg b/build/config/armhf/hd-media.cfg
index 053bd8488..aa1a9205a 100644
--- a/build/config/armhf/hd-media.cfg
+++ b/build/config/armhf/hd-media.cfg
@@ -1,5 +1,5 @@
 FLAVOUR_SUPPORTED = ""
-FLOPPY_SIZE = 40000
+FLOPPY_SIZE = 50000
 
 GZIPPED = .gz
 EXTRANAME = hd-media/
diff --git a/build/config/armhf/netboot.cfg b/build/config/armhf/netboot.cfg
index 381010ce1..2ca092874 100644
--- a/build/config/armhf/netboot.cfg
+++ b/build/config/armhf/netboot.cfg
@@ -10,7 +10,7 @@ EXTRANAME = $(MEDIUM)/
 MANIFEST-INITRD = "netboot initrd"
 MANIFEST-KERNEL = "kernel image to netboot"
 
-FLOPPY_SIZE = 40000
+FLOPPY_SIZE = 50000
 
 GZIPPED = .gz
 
-- 
2.11.0

